config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -5740,3 +5740,8 @@ rubrics.integration.token-secret=12345678900909091234
 # If it is zero, no rotation happens
 # DEFAULT: 30 days
 # lti.advantage.key.rotation.days=30
+
+# OOTB file conversion. This is the poor man's alternative to the fileconversionservice. By default
+# it is limited to files < 10MB in size. You can up that here.
```

Comment:
Maybe some information that setting this to something high could have bad results